keanu-project/src/main/java/io/improbable/keanu/vertices/dbl/DoubleVertex.java

```diff
@@ -49,6 +49,7 @@
 import io.improbable.keanu.vertices.dbl.nonprobabilistic.operators.unary.TakeVertex;
 import io.improbable.keanu.vertices.dbl.nonprobabilistic.operators.unary.TanVertex;
 
+import io.improbable.keanu.vertices.utility.PrintVertex;
```

Comment:
The verticies package is organized by data type and not function. This print would apply to all types so I’d probably put it in the generic package.

Reply:
cool, moved to 
`io.improbable.keanu.vertices.generic.nonprobabilistic`